nilearn/__init__.py

```diff
@@ -52,7 +52,7 @@
 
 # Monkey-patch gzip to have faster reads on large gzip files
 if hasattr(gzip.GzipFile, "max_read_chunk"):
-    gzip.GzipFile.max_read_chunk = 100 * 1024 * 1024  # 100Mb
+    gzip.GzipFile.max_read_chunk = 100 * 1024 * 1024  # type: ignore[attr-defined]
```

Comment:
I have not been able to find any documentation where this is used. Git blame was not very helpful in explaining why this was introduced.

Very tempted to remove it.

Reply:
Try it ?

Reply:
ok but I doubt it will break anything since it may 'only' affect performance, but I could not find where this even exist in the gzip module so I think this won't actually change much.